MSAL/src/instance/MSALDeviceInfoProvider.m

```diff
@@ -81,31 +84,37 @@ - (void)deviceInfoWithRequestParameters:(MSIDRequestParameters *)requestParamete
     if (@available(iOS 13.0, macOS 10.15, *))
     {
         // We are here means canCallSSOExtension is TRUE
+        MSID_LOG_WITH_CTX(MSIDLogLevelInfo, requestParameters, @"GetDeviceInfo: Creating Sso Extension request");
         NSError *requestError;
         MSIDSSOExtensionGetDeviceInfoRequest *ssoExtensionRequest = [[MSIDSSOExtensionGetDeviceInfoRequest alloc] initWithRequestParameters:requestParameters
                                                                                                                                       error:&requestError];
 
         if (!ssoExtensionRequest)
         {
+            MSID_LOG_WITH_CTX_PII(MSIDLogLevelError, requestParameters, @"GetDeviceInfo: Error when creating ssoExtensionRequest: %@", requestError);
             completionBlock(nil, requestError);
             return;
         }
 
         if (![self setCurrentSSOExtensionRequest:ssoExtensionRequest])
         {
+            MSID_LOG_WITH_CTX(MSIDLogLevelError, requestParameters, @"GetDeviceInfo: Cannot start Sso Extension as one is in progress");
             NSError *error = MSIDCreateError(MSIDErrorDomain, MSIDErrorInternal, @"Trying to start a get accounts request while one is already executing", nil, nil, nil, nil, nil, YES);
             completionBlock(nil, error);
             return;
         }
 
+        MSID_LOG_WITH_CTX(MSIDLogLevelInfo, requestParameters, @"GetDeviceInfo: Start querying Sso Extension request with request params");
```

Comment:
Could this be too verbose for info level logs?

Reply:
Updated per comment.

deviceInfo contains extraDict which may contain trackable information, so make it as PII